### PR TITLE
chore: add vue peerDependencies, lint monorepo with `sherif`

### PR DESCRIPTION
**Pull request by sadeghbarati**

### CODE REVIEW

File: packages/vaul-vue/package.json [hunk 1]
```diff
@@ -15,9 +15,10 @@
     "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore",
     "format": "prettier --write src/"
   },
-  "dependencies": {
-    "radix-vue": "^1.2.5",
-    "vue": "^3.4.5"
+  "peerDependencies": {
+    "radix-vue": "^1.2.0",
```
Comment (Elliot-Alexander) [inline packages/vaul-vue/package.json:19]: My thought was to bundle the radix-vue dependency so that it works standalone. Thoughts?

**Comment by Elliot-Alexander** [comment]
I've not used `sherif` before, is it something we should have installed / enforced in CI?

File: packages/vaul-vue/package.json (hunk 1 above)
Comment (sadeghbarati) [reply packages/vaul-vue/package.json:19]: Ow yes you right 👍

**Comment by sadeghbarati** [comment]
> I've not used `sherif` before, is it something we should have installed / enforced in CI?


No, its Rust Tool which check your workspace package.json problem which is not needed to install in CI 

Here the link

https://github.com/QuiiBz/sherif

**Pull request closed without merging**